content/docs/use-cases/ci-and-ml.md

```diff
@@ -0,0 +1,172 @@
+# CI and ML
+
+## Motivation
+
+ML engineers and data scientists face many unique challenges. Some of these
+challenges can be solved by adopting well-established concepts from traditional
+software engineering and development. Probably the most common adoption is
+versioning control. Another adoption, unfortunately less common though, could be unit testing
+the various stages of a data processing pipeline. Here is a non exhaustive list of a few
+other unique challenges:
+
+- Automatically re-train models upon changes.
+- Systematically compare (and report) performance of different versions of a
+  model.
+- Test data correctness when new datasets are introduced.
+- Handle a model training that is computationally intensive.
+
+Combining a version control tool (like `git`) together with `dvc` and `cml`
+opens a wide spectrum of possibilities that can help building better and smarter
+data products. Adding [CI](https://en.wikipedia.org/wiki/Continuous_integration)
+concepts to this triple allows the bulding and **coding** of flows that can help
+tackling the aforementioned problems and many others. In this article, you will
+learn about a few CI use cases utilizing `dvc` and `cml`.
```

Comment:
I removed the last sentence. I would like to keep the existing **Bonuses**. What about the "What's next"? I give a hint to another use case there and I think it's worthy to keep. I will revise it and your feedback would be much appreciated.

Reply:
> I found this paragraph too indirect. I could be brief and to the point, which AFAIK is: 1. We propose CI as a solution for the challenges and 2. This can be done combining Git, DVC, and CML, as will be shown ahead.

In my opinion, this paragraph is already rather short and fits nicely with the introduction. I wouldn't like to replace it with bullet points.

Reply:
OK, no strong opinion about Bonus sections ⚡ just very slightly concerned about consistency in structure among our use cases.

I see the paragraph has been completely rewritten too, so resolving this.